src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -695,13 +695,15 @@ public override unsafe void WriteHeader(string filename, FileHeader* header)
                     PalHelper.ThrowLastError(rc, errorCode, $"Failed to rvn_write_header '{filename}', reason : {((PalFlags.FailCodes)rc).ToString()}");
             }
 
-            public void DeleteAllTempBuffers()
+            public void DeleteAllTempFiles()
             {
                 if (Directory.Exists(TempPath.FullPath) == false)
                     return;
 
-                foreach (var file in Directory.GetFiles(TempPath.FullPath, "*.buffers"))
+                foreach (var file in Directory.GetFiles(TempPath.FullPath))
```

Comment:
Why do we need this? Do we have other files here?

Reply:
Never mind, the temp stream files.

Reply:
Yes. They are supposed to be deleted on close but if the process is aborted (e.g. because of AVE) then they might stay here.